src/views/EmbedView.vue

```diff
@@ -17,13 +17,53 @@ const dispatcher = { dispatcher: 'EmbedView.vue' }
 const store = useStore()
 
 const is3DActive = computed(() => store.state.cesium.active)
+const selectedLayerFeatures = computed(() => store.getters.selectedLayerFeatures)
 
 onBeforeMount(() => {
     store.dispatch('setEmbed', { embed: true, ...dispatcher })
 })
 
 onMounted(() => {
-    log.info(`Map view mounted`)
+    log.info(`Embedded map view mounted`)
+})
+
+// as described by this example on our documentation : https://codepen.io/geoadmin/pen/yOBzqM?editors=0010
+// our embed view should send a message (see https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage)
+// when a feature is selected while in embedded mode, so that the parent can get the selected feature(s) ID(s)
+watch(selectedLayerFeatures, () => {
+    const targetWindow = parent ?? window.parent ?? window.opener ?? window.top
+    if (!targetWindow) {
+        log.debug(
+            'Embed view loaded as root document of a browser tab, cannot communicate with opener/parent'
+        )
+        return
+    }
+    // if no features are selected, nothing to do
+    if (selectedLayerFeatures.value.length === 0) {
+        return
+    }
+    log.debug('sending information about selected features to iframe parent')
+    // from what I can understand from the codepen, one event is fired per feature with a structured response
+    selectedLayerFeatures.value.forEach((feature) => {
+        targetWindow.postMessage(
+            {
+                // see codepen above, for backward compatibility reasons we need to use the same type as mf-geoadmin3
+                type: 'gaFeatureSelection',
+                payload: {
+                    layerId: feature.layer.id,
+                    featureId: feature.id,
+                    // if we want to expose more stuff from our features (EGID, EWID, etc...), it should come here...
+                },
+            },
+            // meaning anyone, any host, can receive this event when adding our app as embedded on their website,
+            // so let's be cautious with what we add to the payload
+            '*'
+        )
+        // mf-geoadmin3 was also sending the same feature in a different unstructured/string format "layerId#featureId"
+        // but this comment here https://github.com/geoadmin/mf-geoadmin3/blob/6a7b99a2cc9980eec27b394ee709305a239549f1/src/components/tooltip/TooltipDirective.js#L661-L668
```

Comment:
I proposed to move that code to an api file and out of the EmbedView component.
Some user uses the normal view in iframe and I'm not if we should also add this to the normal map view. But simply for code organization would be better to just have a function call here, or use a composable at best.